src/DataProtection/DataProtection/src/DataProtectionBuilderExtensions.cs

```diff
@@ -408,6 +412,7 @@ public static IDataProtectionBuilder ProtectKeysWithDpapi(this IDataProtectionBu
         /// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706794(v=vs.85).aspx
         /// for more information on DPAPI-NG. This API is only supported on Windows 8 / Windows Server 2012 and higher.
         /// </remarks>
+        [SupportedOSPlatform("windows")]
```

Comment:
@terrajobst Should we mark the APIs that are "supported on Windows 8 / Windows Server 2012 and higher" as `[SupportedOSPlatform("windows8.0")]` since 7.0 is our defacto baseline?